lib/widgets/dialogs.dart

```diff
@@ -2,32 +2,39 @@ import 'package:flutter/material.dart';
 import 'package:bars_frontend/main.dart';
 import 'package:bars_frontend/utils.dart';
 
-Future<dynamic> asyncInputDialog(
-    BuildContext context,
-    MyHomePageState homePageState,
-    MapEntry<String, dynamic> feature) async {
+Future<dynamic> asyncInputDialog(BuildContext context,
+    MyHomePageState homePageState, MapEntry<String, dynamic> feature) async {
   return showDialog<dynamic>(
     context: context,
     barrierDismissible: true,
     builder: (BuildContext context) {
       return Container(
         child: SimpleDialog(
+          titlePadding: EdgeInsets.fromLTRB(0, 40, 0, 0),
+          title: Text(
+            "Please enter your information",
+            textAlign: TextAlign.center,
+          ),
           children: <Widget>[
             Padding(
-              padding: const EdgeInsets.all(50.0),
-              child: Column(
-                children: <Widget>[
-                  MyDialogContent(homePageState, feature),
-                  Container(height: 50),
-                  SimpleDialogOption(
+              padding: EdgeInsets.fromLTRB(40, 0, 40, 20),
+              child: Column(children: <Widget>[
+                MyDialogContent(homePageState, feature),
+                Container(
+                  height: 40,
+                  width: 100,
```

Comment:
I don't think the button should scale according to screen size. 40/100 is a standard button size.

Reply:
I'm find with keeping as is. Either way it would be bad for a tiny screen, and this prototype doesn't care about tiny screens in the first place. For big screens, scaling would be weird too.